### DISCUSSION TITLE
CA importcert problem - server cert with multiple SAN shows import error

### DISCUSSION THREAD

**Discussion by pandel** [Keyfactor/ejbca-ce#129] [General]
Hi!

I try to import an old cert via `ejbca ca importcert`, but I get the following error message:

```
Certificate Profile: SERVER
End Entity Profile: EMPTY
SubjectAltName: dNSName=host1.machine.net, dNSName=host1, dNSName=host2.machine.net, dNSName=host2, iPAddress=10.XX.XX.132, iPAddress=10.XX.XX.110
Type: 1
User does not fulfill end entity profile: Wrong number of DNSNAME fields in Subject Alternative Name.
```

I tried to copy the SERVER profile to a custom one and modify the X.509v3 Extention values, but nothing worked so far.

How do I have to modify the SERVER certificate profile to successfully import this certificate?

Regards,
Holger

**Comment by primetomas** [comment]
This is the end entity profile that causes the error not the certificate profile.

The fixed, hardcoded profiles, as SERVER and EMPTY should be used as templates, just as you did. You have to import using your modified end entity profile instead of the fixed one (EMPTY). In your end entity profile you must include at least 4 dnsNames.

**Reply by pandel** [reply]
@primetomas 
Thanks for your answer. I thought that EMPTY would work kind of like a wildcard, regardless of the structure of the certificate. Ok, I'll figure it out!
